src/amo/components/AddonMoreInfo/index.js

```diff
@@ -57,6 +49,75 @@ export class AddonMoreInfoBase extends React.Component {
       );
     }
 
+    return this.renderDefinitions({
+      homepage,
+      supportUrl,
+      version: addon.current_version.version,
+      versionLastUpdated: i18n.sprintf(
+        // translators: This will output, in English:
+        // "2 months ago (Dec 12 2016)"
+        i18n.gettext('%(timeFromNow)s (%(date)s)'), {
+          timeFromNow: i18n.moment(addon.last_updated).fromNow(),
+          date: i18n.moment(addon.last_updated).format('ll'),
+        }
+      ),
+      versionLicenseLink: addon.current_version.license ? (
+        <a
+          className="AddonMoreInfo-license-link"
+          href={addon.current_version.license.url}
+        >
+          {addon.current_version.license.name}
+        </a>
+      ) : null,
+      privacyPolicyLink: addon.has_privacy_policy ? (
+        <Link
+          className="AddonMoreInfo-privacy-policy-link"
+          href={`/addon/${addon.slug}/privacy/`}
+        >
+          {i18n.gettext('Read the privacy policy for this add-on')}
+        </Link>
+      ) : null,
+      eulaLink: addon.has_eula ? (
+        <Link
+          className="AddonMoreInfo-eula-link"
+          href={`/addon/${addon.slug}/eula/`}
+        >
+          {i18n.gettext('Read the license agreement for this add-on')}
+        </Link>
+      ) : null,
+      addonId: addon.id,
+      versionHistoryLink: (
+        <a
+          className="AddonMoreInfo-version-history-link"
+          href={`/addon/${addon.slug}/versions/`}
+        >
+          {i18n.gettext('See all versions')}
+        </a>
+      ),
+      versionDevChannelLink: (
+        <a
+          className="AddonMoreInfo-dev-channel-link"
+          href={`/addon/${addon.slug}/versions/beta`}
+        >
+          {i18n.gettext('See experimental versions')}
```

Comment:
I've mentioned this elsewhere but I think on the new front-end we should be referring to these as "Beta versions" not "Development" or "Experimental". It's a confusing name for users and the intent of these versions is bleeding edge features/fixes for adventurous users–exactly what a beta is 😄

So I think this could be:

```html
<dt>Version History</dt>
<dd>See all versions</dd>
<dt>Beta Version</dt>
<dd>See all beta versions</dd>
```